sign/cosi/cosi_test.go

```diff
@@ -28,15 +28,31 @@ func (m *cosiSuite) RandomStream() cipher.Stream { return m.r }
 var testSuite = &cosiSuite{edwards25519.NewBlakeSHA256Ed25519(), blake2xb.New(nil)}
 
 func TestCoSi(t *testing.T) {
-	testCoSi(t, 2, 0)
-	testCoSi(t, 5, 0)
-	testCoSi(t, 5, 2)
-	testCoSi(t, 5, 4)
+	message := []byte("Hello World Cosi")
+	testCoSi(t, 2, 0, message)
+	testCoSi(t, 2, 0, message)
```

Comment:
why repeat this twice ?

Reply:
Yeah probably I should have misclicked a ctrl+v